lib/modules/dosomething/dosomething_reportback/dosomething_reportback.module

```diff
@@ -523,17 +523,8 @@ function dosomething_reportback_view_entity($entity) {
     $fid = array_pop($entity->fids);
   }
 
-
-  // @TODO Can we templatize/DRY some of this? It's fairly similiar code in multiple places.
-  $disable_reactions = dosomething_kudos_disable_reactions($entity->nid);
-
-  if (! $disable_reactions) {
-    $kudos = [];
-    $kudos['rbid'] = $entity->rbid;
-    $kudos['disable_reactions'] = $disable_reactions;
-    $kudos['allowed_reactions'] = [dosomething_kudos_get_term_id_by_name('heart')];
-    $kudos['reaction_totals'] = ['heart' => dosomething_kudos_get_total_for_file_by_term_name($entity->rbid, 'heart')];
-    $kudos['existing_kids'] = dosomething_kudos_get_kudos_for_user_by_file($entity->rbid);
+  if (! dosomething_kudos_disable_reactions($entity->nid)) {
```

Comment:
maybe set:

```php
$kudos = [];
```
above the conditional otherwise you'll have an error here: https://github.com/DoSomething/phoenix/pull/6689/files#diff-6cd2bd158ecf1b2419b491fc3dae7ceaR601 if `$kudos` is not set @_@